LEAF_Request_Portal/form.php

```diff
@@ -3061,6 +3066,24 @@ public function query($inQuery)
             }
         }
 
+        if ($joinRecordResolutionBy) {
+            require_once 'VAMC_Directory.php';
+            $dir = new VAMC_Directory;
+
+            $res2 = $this->db->prepared_query('SELECT recordID, userID as resolvedBy FROM action_history
+                                                LEFT JOIN dependencies USING (dependencyID)
+                                                LEFT JOIN actions USING (actionType)
+                                                WHERE recordID IN (' . $recordIDs . ')
+                                                and stepID > 0
+                                                ORDER BY time', array());
+
+            foreach ($res2 as $item) {
+                $user = $dir->lookupLogin($item['resolvedBy']);
+                $nameResolved = isset($user[0]) ? "{$user[0]['Fname']} {$user[0]['Lname']}" : $item['resolvedBy'];
```

Comment:
As per the rest of our code base, we should be listing names as "lastName, firstName" as it does when searching users and displaying them within the User Access Group admin section.

Reply:
fixed